includes/modules/shipping_estimator.php

```diff
@@ -136,7 +136,7 @@ function shipincart_submit() {
                 ],
                 'country_id' => STORE_COUNTRY,
                 'zone_id' => $state_zone_id,
-                'format_id' => zen_get_address_format_id((int)$_POST['zone_country_id'] ?? 0),
```

Comment:
Since this is the `else` to line 86 which triggers when this POST value is set, this is almost certainly never going to exist.
Would a better solution be to replace POST[zone_country_id] here with `$_SESSION['cart_country_id']` or `STORE_COUNTRY` as used in the rest of this `else` clause?

Reply:
This bug is in the final clause:
![image](https://github.com/zencart/zencart/assets/4391026/33e954c8-2fcc-4dc8-8cc1-cef678173e48)

I think STORE_COUNTRY is the most appropriate replacement.